.changeset/nervous-guests-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/commercial': minor
+---
+
+Upgrade to prebid.js v8
```

Comment:
There are two changesets for this change - could we remove one of them?

Reply:
oops 🙃